src/hotspot/share/gc/shenandoah/shenandoahPacer.hpp

```diff
@@ -29,8 +29,24 @@
 #include "gc/shenandoah/shenandoahPadding.hpp"
 #include "gc/shenandoah/shenandoahSharedVariables.hpp"
 #include "memory/allocation.hpp"
+#include "runtime/task.hpp"
 
 class ShenandoahHeap;
+class ShenandoahPacer;
+
+
+// Periodic task to notify blocked paced waiters.
+class ShenandoahPeriodicPacerNotify : public PeriodicTask {
+public:
+  explicit ShenandoahPeriodicPacerNotify(ShenandoahPacer* pacer) :
+    PeriodicTask(PeriodicTask::min_interval),
+    _pacer(pacer) { }
+
+  void task() override;
+private:
+  ShenandoahPacer* _pacer;
```

Comment:
Move this declaration at the beginning and make `_pacer` const, while you are at it?